smoke-tests/src/test/groovy/io/opentelemetry/smoketest/TomcatSmokeTest.groovy

```diff
@@ -5,11 +5,18 @@
 
 package io.opentelemetry.smoketest
 
+import java.time.Duration
+
 abstract class TomcatSmokeTest extends AppServerTest {
 
   protected String getTargetImagePrefix() {
     "ghcr.io/open-telemetry/opentelemetry-java-instrumentation/smoke-test-servlet-tomcat"
   }
+
+  @Override
+  protected TargetWaitStrategy getWaitStrategy() {
```

Comment:
I saw connection failures and thought it could be Armeria-related, but realized that the tests were running before the server started. I don't think there's anything in this PR that would slow down startup time but maybe there is? Or just at a borderline that I'm unluckily breaking